components/bio-formats-plugins/src/loci/plugins/util/ROIHandler.java

```diff
@@ -50,8 +48,9 @@
 import loci.formats.meta.IMetadata;
 import loci.formats.meta.MetadataStore;
 import loci.formats.ome.OMEXMLMetadata;
+import ome.model.units.Length;
```

Comment:
I've corrected for that, trying to fix a few other shape parameters for Label Text.